### PR TITLE
Edit profile updates

### PR DESCRIPTION
**Pull request by felixportuphy**

### CODE REVIEW

File: cypress-open.json [hunk 1]
```diff
@@ -13,4 +13,5 @@
   "retries": 2,
   "scrollBehavior": "center",
   "chromeWebSecurity": false
+
```
Comment (JerryHukpati) [inline cypress-open.json:16]: remove empty line

File: cypress/support/commands.js [hunk 2]
```diff
@@ -108,59 +108,29 @@ Cypress.Commands.add('gotoConfirmCodePage', () => {
...
-Cypress.Commands.add('gotoEditProfilePage', () => {
-  cy.login().then(() => {
-    cy.get('.btn-edit').contains('Edit Profile').click()
-  })
-})
+Cypress.Commands.add('login', (email, password) => {
+  cy.intercept({
+    method: 'POST',
+    url: 'https://**/auth/check-user/',
+  }).as('existingUser')
```
Comment (JerryHukpati) [inline cypress/support/commands.js:115]: why new login custom command

File: cypress/support/commands.js [hunk 3]
```diff
@@ -108,59 +108,29 @@ Cypress.Commands.add('gotoConfirmCodePage', () => {
...
-      }
-    }
-    cy.selectYear(yearName)
-  })
+  cy.get('[data-cy="loginEmail"]').type(email)
+  cy.get('#submit_button').contains('Continue').click()
+  cy.wait('@existingUser')
+  cy.get('[data-cy="signinPassword"]').type(password)
+  cy.get('#siginBtn').contains('Sign in').click()
 })
```
Comment (JerryHukpati) [inline cypress/support/commands.js:122]: refer to above comment

**Review by JerryHukpati — CHANGES_REQUESTED** [review]

File: cypress-open.json (hunk 1 above)
Comment (felixportuphy) [reply cypress-open.json:16]: Fixed

File: cypress/support/commands.js (hunk 3 above)
Comment (felixportuphy) [reply cypress/support/commands.js:122]: Fixed

**Review by JerryHukpati — APPROVED** [review]

**Pull request closed without merging**